.eslintrc

```diff
@@ -3,23 +3,14 @@
   "plugins": [ "react" ],
   "env": {
     "es6": true,
-    "browser": true,
-  },
-  "ecmaFeatures": {
-    "templateStrings": true,
-    "arrowFunctions": true,
-    "modules": true,
-    "blockBindings": true,
-    "forOf": true,
-    "jsx": true
+    "browser": true
   },
   "rules": {
     "no-unused-vars": 1,
     "react/jsx-uses-vars": 1,
     "no-var": 2,
     "new-cap": 0,
     "quotes": [1, "single", "avoid-escape"],
-    "semi": 1,
-    "indent": [2, 2]
```

Comment:
Removing this is annoying ... as the rule is generally ok, but there are a bunch of exceptions which probably need to be added to get it working. (help ...?)

An alternative is to not ugprade eslint to 4.19.  I chose that version because it is the same as being used on git-task-list, and IMO we should aim at having the same in our JS repos.

Reply:
Airbnb style actually uses 4.9 as peer dependency. I really want to downgrade to that matching version :( too. Waiting for @blazeu .

Reply:
Was using 4.19 as i though, later is better.

Reply:
> Airbnb style actually uses 4.9 as peer dependency.

No it doesnt.
It uses `eslint: '^4.9.0'`, which in case you didnt know means any version of 4, after that.

To check what a JS semvar spec means, https://semver.npmjs.com/ is very helpful.

Reply:
4.9 seems to work fine and no additional overhead.

Reply:
I know it would work with 4.19 with no dependency error. But it add additional overhead.